oop/src/views/MoveInfoPage.vue

```diff
@@ -362,17 +379,35 @@ export default {
     getMinAddressLines() {
       return MIN_ADDRESS_LINES;
     },
+    setFieldsToNull() {
+      // Remove all address lines
+      for (let i=0; i<this.addressLines.length-1; i++) {
+        this.addressLines.pop();
+      }
+      // Set first address line to null
+      if (document.getElementById('inputAddress line 1')){
+        this.addressLines[0].value = null;
+        document.getElementById('inputAddress line 1').value = null;
+      }
+      // Set province to null
+      this.province = null;
+      if (document.getElementById('province')){
+        document.getElementById('province').value = '';
+      }
```

Comment:
I think we want to avoid DOM queries as much we can in the Vue components. Can this be written as the following instead?

```javascript
if (this.province) {
  this.province = '';
}
```

The DOM element shouldn't need to be set since it's binding on the `this.province` variable, no?